### DISCUSSION TITLE
Section 1: Lesson 46. Incorrect code example

### DISCUSSION THREAD

**Discussion by PavelNaydanov** [Cyfrin/assembly-evm-opcodes-and-formal-verification-course#23] [Q&A] [answered]
Thank you for your course. It is magical. Please don't think that I'm being picky.
You need to check "Written lesson" section for lesson 46. 

Misprint
> contructor payable () {} 

Fixed 
> constructor() payable {}

**Comment by EngrPips** [comment, marked as answer]
Hey @PavelNaydanov, Thanks for the contribution but you can actually make a pull request to get the changes reflected on the written lesson. Here is the repo for all Cyfrin Courses https://github.com/Cyfrin/Updraft
